main.go

```diff
@@ -93,11 +94,34 @@ func NewService(ctx context.Context) *Service {
 
 	logger := log.New()
 	logger.SetFormatter(&log.JSONFormatter{})
-	logger.SetOutput(os.Stdout)
 	logger.SetLevel(log.InfoLevel)
+
+	fileLoggerHook, err := lumberjackrus.NewHook(
+		&lumberjackrus.LogFile{
+			Filename: "nwc.general.log",
```

Comment:
Some questions:
- The general logs have no options (like MaxAge), is this ok?
- Currently general logs will show warning and fatal messages. I think this is a bit of a strange combination. I wonder if we should just put all in one file?
- Is it intended that nwc.general.log is in the main directory but the others are in "log"?
- there is a bit of inconsistency in naming with dots and dashes. I think it would be good to use the same convention or just have one log file, what do you think?

Reply:
Done